tests/locallib_test.php

```diff
@@ -80,6 +81,7 @@ class locallib_test extends \advanced_testcase {
      * @var int
      */
     public $assign;
+    private int $courseid;
```

Comment:
Could you add a phpdoc comment to line 84 to keep the linter quiet?

Reply:
Oops! I've added that now.